app/src/views/OperationalLearnings/Filters/index.tsx

```diff
@@ -0,0 +1,110 @@
+
+import i18n from './i18n.json';
+
+type Option = {
+    key: string;
+    label: string;
+};
+
+function Filters() {
+    const strings = useTranslation(i18n);
+    const options: Option[] = [];
+    const keySelector = (option: Option) => option.key;
+    const labelSelector = (option: Option) => option.label;
+    const {
+        rawFilter,
+        setFilter,
+        setFilterField,
+    } = useFilterState<{
+        region: RegionOption['key'],
+        country: number,
+        disasterType:number,
+        startDateBefore?: string,
+        bySector?:number,
+        byComponent?:number,
+        startDate?:string
+    }>({
+        filter: {
+            region: 0,
+            country: 0,
+            disasterType: 0,
+        },
+    });
+    const handleExportClick = useCallback(() => setFilter({}), [setFilter]);
+
+    return (
+        <Container
+            contentViewType="vertical"
+            withGridViewInFilter
+            filters={(
+                <>
+                    <RegionSelectInput
+                        placeholder={strings.filterRegionsPlaceholder}
+                        name="region"
+                        onChange={setFilterField}
+                        value={rawFilter.region}
+                    />
+                    <CountrySelectInput
+                        placeholder={strings.filterCountryPlaceholder}
+                        name="country"
+                        value={rawFilter.country}
+                        onChange={setFilterField}
+                    />
+                    <DisasterTypeSelectInput
+                        placeholder={strings.filterDisasterTypePlaceholder}
+                        name="disasterType"
+                        value={rawFilter.disasterType}
+                        onChange={setFilterField}
+                    />
+                    <SelectInput
+                        placeholder={strings.filterBySectorPlaceholder}
+                        name="bySector"
+                        options={options}
+                        keySelector={keySelector}
+                        labelSelector={labelSelector}
+                        value={rawFilter.bySector}
+                        onChange={setFilterField}
+                    />
+                    <SelectInput
+                        placeholder={strings.filterByComponentPlaceholder}
+                        name="byComponent"
+                        options={options}
+                        keySelector={keySelector}
+                        labelSelector={labelSelector}
+                        value={rawFilter.byComponent}
+                        onChange={setFilterField}
+                    />
+                    <DateInput
+                        name="startDateFrom"
+                        onChange={setFilterField}
+                        value={rawFilter.startDate}
+                    />
+                    <KeywordSearchSelectInput />
```

Comment:
This is a specific component defined to do a `search` in IFRC Go. Let's not use these here. Let's use `TextInput` component instead.